src/en/404.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: base
+title: Not found
+tags: 'warning'
+permalink: '/{{ locale }}/404.html'
+---
+
+<section class="section text-center">
+  <h1 class="h1">404</h1>
+  <p class="standout">The page you are looking for is not here (anymore).</p>
```

Comment:
```suggestion
  <p class="standout">The page you are looking for is not here.</p>
```

Suggestion: Perhaps we remove `(anymore)` here. While it might be true of some old pages which no longer exist on the new site (those that haven't had 301 redirects put in place), it won't always be true of others, e.g. https://develop.ceph.io/bogus-page — perhaps best to just say "not here"?

Reply:
Yeah, that seems fair. I've updated.